src/autocompletion.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * This file is part of OpenTTD.
+ * OpenTTD is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 2.
+ * OpenTTD is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file autocompletion.h Generic auto-completion engine. */
+
+#ifndef AUTOCOMPLETION_H
+#define AUTOCOMPLETION_H
+
+#include "textbuf_type.h"
+
+class AutoCompletion {
+protected:
+	Textbuf*                 textbuf;
```

Comment:
we no longer generally bother with the alignment stuff, especially not multiple "sections"

Also, `*` goes on the variable

Reply:
Done

Reply:
Not everywhere :)

Reply:
🤦

Reply:
Looked through the code one more time and fix the only place I noticed.